test/node/ssr.test.tsx

```diff
@@ -23,14 +22,10 @@ function App() {
   return <DataGrid columns={columns} rows={rows} />;
 }
 
-// biome-ignore lint/suspicious/noSkippedTests: <explanation>
-test.skip('basic server-side rendering (SSR) support', () => {
+test('basic server-side rendering (SSR) support', () => {
   // make sure we're not running in the JSDOM environment
   expect(globalThis.window).toBeUndefined();
 
   const html = renderToString(<App />);
   expect(html).not.toHaveLength(0);
-
-  // make sure `@testing-library/jest-dom` does not polyfill `window.CSS`
-  expect(typeof CSS).toBe('undefined');
```

Comment:
https://github.com/adazzle/react-data-grid/commit/b9e3344cd53c96b45a4fa3dbb6c5d0a9425f690e
Seems like this hasn't been needed for a while.